### ISSUE TITLE
Spreadsheets in Lieu of "I'm Offering"

### ISSUE THREAD

**Issue by broman** [rorylombardi/tradingpost#2]
Many traders offer a spreadsheet of Pokemon rather than specific Pokemon. [See example](https://docs.google.com/spreadsheets/d/1tnJRs-6xNvEQUibTx6C_UJUET_hiAgoxkRnv0JvoLtU/edit#gid=1086466421). In the future as this project gains attention I would expect this feature to be requested.

**Comment by rorylombardi** [comment]
Thanks for filing this feature request! You're able to view your own listings today via the logged-in menu -> My Listings: is that what you're looking for?

**Comment by broman** [comment]
No, I'm suggesting that user's should be able to link a spreadsheet of Pokemon they have for trade rather than being forced to list specific ones.

**Comment by rorylombardi** [comment]
Oh, I see: you want to have an alternative to the "looking for" field. I'll rename this issue to "Listing notes field".
